azure-commons-plugin/pom.xml

```diff
@@ -15,6 +15,12 @@
     <description>Common APIs for Azure related Jenkins plugins</description>
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Azure+Commons+Plugin</url>
 
+    <properties>
+        <enableAzureApiTrace>false</enableAzureApiTrace>
+        <enablePageDecorator>false</enablePageDecorator>
```

Comment:
what's this?

Reply:
When one of the pages in Jenkins being rendering, the page decorator can be notified and decide whether to send a ping event to our AI.